### DISCUSSION TITLE
ROS2 - what is required from me?

### DISCUSSION THREAD

**Discussion by peredwardsson** [foxglove/community#77] [Q&A] [answered]
Hello,
I am running ROS2 Foxy on my robot, who I can access on my network, and whose topic show up when running `ros2 topic list`. However, no topics (or indeed any data at all) show up inside foxglove studio, which makes me wonder if I am missing something here.
I am running in a docker container with Foxy, where I followed the install instructions from https://foxglove.dev/download. I start `foxglove-studio` from inside the docker, open a panel for raw messages, and input a message which I can view via `ros2 topic echo`, for instance `/localization/odometry/filtered`, which is the output of my EKF node, of type `nav_msgs/msg/Odometry`. Am I doing something wrong? 

Error message from terminal:
`[1179:1213/122405.929485:ERROR:bus.cc(393)] Failed to connect to the bus: Could not parse server address: Unknown address type (examples of valid types are "tcp" and on UNIX "unix")`
Robot has an IPv6 address - is that a problem?

**Comment by jhurliman** [comment, marked as answer]
It is a problem for now, yes. Our [RTPS implementation](https://github.com/foxglove/rtps) only supports IPv4 currently. I created an issue (https://github.com/foxglove/rtps/issues/13) to track IPv6 support.
